match/spec/encryption/openssl_spec.rb

```diff
@@ -40,5 +40,23 @@
         @e.encrypt_files
       end.to raise_error("No password supplied")
     end
+
+    it "doesn't raise an exception if no env var is supplied but custom password is" do
+      ENV["MATCH_PASSWORD"] = ""
```

Comment:
This will prevent the `ENV` value from carrying over between tests

```suggestion
      stub_const('ENV', { "MATCH_PASSWORD" => "" }
```